### PR TITLE
store pages in bitbox

### PR DESCRIPTION
**Pull request by pepyakin**

### CODE REVIEW

**Comment by pepyakin** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/thrumdev/nomt/237?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#237** <a href="https://app.graphite.dev/github/pr/thrumdev/nomt/237?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#236** <a href="https://app.graphite.dev/github/pr/thrumdev/nomt/236?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#235** <a href="https://app.graphite.dev/github/pr/thrumdev/nomt/235?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @pepyakin and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: nomt/src/store.rs [hunk 1]
```diff
@@ -131,17 +133,14 @@ impl Transaction {
     }
 
     /// Write a page to storage in its entirety.
-    pub fn write_page<V: AsRef<[u8]>>(&mut self, page_id: PageId, value: V) {
-        let cf = self.shared.db.cf_handle(PAGES_CF).unwrap();
-        self.batch
-            .put_cf(&cf, page_id.length_dependent_encoding(), value);
+    pub fn write_page(&mut self, page_id: PageId, page: &Vec<u8>, page_diff: PageDiff) {
```
Comment (rphmeier) [inline nomt/src/store.rs:136]: `&[u8]` is better than `&Vec<u8>`!

Comment (pepyakin) [reply nomt/src/store.rs:136]: We can agree on that and that's a verbatim copy. We should make a clean-up sweep afterwards.

**Comment by pepyakin** [comment]
### Merge activity

* **Aug 12, 1:07 PM EDT**: @pepyakin started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/thrumdev/nomt/237).

**Pull request merged**